app/core/config.py

```
@@ -16,52 +42,122 @@ class BaseConfig:
 
-class TestConfig(BaseConfig):
-    # base
-    ENV: str = "test"
-    DB_URL = "sqlite+aiosqlite:///:memory:"
-    SYNC_DB_URL = "sqlite:///:memory:"
+configs = Configs()
 
+if ENV == "prod":
+    pass
+elif ENV == "stage":
+    pass
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)